lib/google-civic/request.rb

```diff
@@ -2,8 +2,8 @@
 
 module GoogleCivic
   module Request
-    def get(path,params={})
-      response = connection.get do |request|
+    def get(path,params={},connection_opts={})
+      response = connection(connection_opts).get do |request|
         request.url path
         params.each { |name, value| request.params[name] = value }
         request.params['key'] = @key
```

Comment:
Is the library we forked using 4-space indentation? Gross. Can we standardize the whole project (or at the least files being touched if that would balloon the PR too much) at 2 spaces?